src/components/Banner/Delegation/DelegationBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import './DelegationBanner.css'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import React, { useEffect, useState } from 'react'
+
+import Delegate from '../../Icon/Delegate'
+import Banner from '../Banner'
+
+type DelegationBannerProps = {
+  isVisible: boolean
+}
+
+const HIDE_DELEGATE_BANNER_KEY = 'org.decentraland.governance.delegate_banner.hide'
+
+function DelegationBanner({ isVisible }: DelegationBannerProps) {
+  const t = useFormatMessage()
+  const [showBanner, setShowBanner] = useState(false)
+
+  useEffect(() => {
+    const showSubsBanner = localStorage.getItem(HIDE_DELEGATE_BANNER_KEY) != 'true'
+    setShowBanner(showSubsBanner)
+  }, [])
+
+  return (
+    <div className="DelegationBanner__Container">
+      <Banner
+        isVisible={showBanner && isVisible}
```

Comment:
Why don't 

```
const isVisible = localStorage.getItem(HIDE_DELEGATE_BANNER_KEY) != 'true'

...

<Banner isVisible={isVisible} />
```

Reply:
Remember that the banner is also hidden depending on whether there is an active search or not.

In `index.tsx` the banner is called this way: 

```html
<div className="OnlyDesktop">
  <DelegationBanner isVisible={!searching} />
</div>
```

Reply:
You are right. What I wanted to say is that you can remove the use effect and just store the localStorage item check in a variable and use that to pass to the isVisible prop.

`isVisible={isVisible && !hasHideDelegateBannerKey}`